MIGRATION.md

```diff
@@ -0,0 +1,114 @@
+# Migrating from `dp-frontend-renderer` to `dp-renderer`
+
+## Migrating assets and models
+
+The frontend service is responsible for setting up the assets binary and source path.
+
+You should also store all assets and models within the application itself with the following structure:
+
+```md
+.
+├── asset                    # relevant templates & localisations from dp-frontend-renderer
+│   ├── templates          
+│   ├── localisations  
+|   |   ├──  service.en.toml
+|   └── └──  service.cy.toml
+└── model                    # relevant models from dp-frontend-models
+```
+
+## Updating the Makefile
+
+In order for `dp-renderer` to work correctly once the assets have been migrated over, we use `go-bindata` to generate a combined assets source file.
+
+Update the frontend app's `Makefile` with the following new commands so that `go-bindata` will generate this file:
+
+```Makefile
+.PHONY: get-renderer-version
+get-renderer-version:
+ $(eval APP_RENDERER_VERSION=$(shell grep "github.com/ONSdigital/dp-renderer" go.mod | cut -d ' ' -f2 ))
+
+.PHONY: fetch-renderer-lib
+fetch-renderer-lib: get-renderer-version
+ $(eval CORE_ASSETS_PATH = $(shell go get github.com/ONSdigital/dp-renderer@$(APP_RENDERER_VERSION) && go list -f '{{.Dir}}' -m github.com/ONSdigital/dp-renderer))
```

Comment:
Maybe not something for now but this possibly would break in a v2 release of dp-renderer. 

e.g. `get-renderer-version` would find `github.com/ONSdigital/dp-renderer/v2 v2.2.3` in the go.mod file and then split it and give you just the version part which works great. However, the `fetch-renderer-lib` would then do something like `go get github.com/ONSdigital/dp-renderer@v2.2.3` where I think (but I may be wrong) it needs to be `go get github.com/ONSdigital/dp-renderer/v2@v2.2.3` So possibly need a second var to store the ` `/`v2`/`v3` part. But, not 100% sure on this. 

We could leave this until it is an issue

Reply:
Yeah this is a good point I'd not considered - I'm not really sure what the best way forward is, but as you say I think it might be okay to leave for now as it won't be an issue for a while yet